grammar.js

```diff
@@ -636,12 +631,19 @@ module.exports = grammar({
       ']'
     ),
 
-    dictionary: $ => seq(
+    dictionary: $ => prec(PREC.dictionary, seq(
```

Comment:
This precedent disambiguates between dictionaries and lists. We give dictionaries a slightly higher precedent so the sequence of `**` is not mistakingly parsed as `list_splat * ...`.